bentoml/_internal/server/cli/api_server.py

```diff
@@ -69,6 +70,10 @@ def main(
     \b
     This is an internal API, users should not use this directly. Instead use `bentoml serve <path> [--options]`
     """
+    import logging
+
+    logger = logging.getLogger(__name__)
+    logger.info(f"Starting BentoML API server {worker_id}...")
```

Comment:
```suggestion
    logger.debug(f"Starting BentoML API server {worker_id}...")
```
Should be a debug message.